src/components/v2/Layout/Sidebar/index.tsx

```diff
@@ -41,16 +45,15 @@ export const SidebarUi: React.FC<ISidebarProps> = ({ account }) => {
             <LogoDesktop css={styles.logo} />
             <LogoNoText css={styles.logoClosed} />
           </Toolbar>
-
           <List css={styles.list}>
-            {menuItems.map(({ href, icon, text }) => (
-              <ListItemButton key={text} component="li" css={styles.listItem} disableRipple>
+            {menuItems.map(({ href, icon, id }) => (
+              <ListItemButton key={id} component="li" css={styles.listItem} disableRipple>
                 <NavLink to={href} activeClassName="active-menu-item">
                   <ListItemIcon css={styles.listItemIcon}>
                     <Icon name={icon} />
                   </ListItemIcon>
                   <Typography variant="body2" css={styles.listItemText}>
-                    {text}
+                    {t(`layout.menuItems.${id}`)}
```

Comment:
Same thing as previous comment about needing to include the text itself inside each menu item.